main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as data_file:
+        data = json.load(data_file)
+    for player in data:
+        race_data = data[player]["race"]
+        race, _ = Race.objects.get_or_create(
+            name=race_data["name"],
+            description=race_data["description"]
+        )
+
+        for skill in race_data["skills"]:
+            Skill.objects.get_or_create(
+                name=skill["name"],
+                bonus=skill["bonus"],
+                race_id=race.id
+            )
+
+        guild_data = data[player]["guild"] if data[player]["guild"] else None
+        if guild_data:
+            Guild.objects.get_or_create(
+                name=guild_data["name"],
+                description=guild_data["description"]
+            )
+            guild = Guild.objects.get(name=guild_data["name"])
+
+            Player.objects.get_or_create(
```

Comment:
This is code duplication. 
Define guild as None, and then do similarly as for race: 
guild, _ = Guild.objects.get_or_create(...